mobile/src/reducers/organization/organization.epics.ts

```diff
@@ -1,5 +1,6 @@
 import { ActionsObservable } from 'redux-observable';
```

Comment:
Move `$errorEpic$` to the separate file `error-load-failed.epic.ts` and rename to `errorLoadFailedEpic$`